ext/standard/tests/file/fnmatch_maxpathlen.phpt

```diff
@@ -0,0 +1,14 @@
+--TEST--
+Test fnmatch() function : warning filename or pattern exceeds maxpathlen
```

Comment:
This test needs a `SKIPIF` section as this function isn't always available :)

Reply:
If other than that all is good, I'll add this one and merge it then...

Reply:
@petk Go ahead.